xtend-website/community.html

```diff
@@ -5,114 +5,121 @@
 <div id="header_wrapper">
 	<div class="container"> 
 		<div class="flexslider image-slider">
-		  <div class="row">
-			<div class="span5 team">
-			  <a href="https://groups.google.com/group/xtend-lang" class="anchor-in-div" title="The Xtend Google Group is the first source for getting answers in case you got stuck. The community is very friendly. Please ask any questions clearly and with sufficient information attached."></a>
-			  <div class="row">
-			    <div class="span1">
-			      <img src="{{ site.baseurl }}/images/discussion-white.png" alt="image" class="image_left">
-			    </div>
-			    <div class="span4">
-			      <h3 >Discussion Group</h3>
-					<p></p>
-				</div>
-			  </div>
-			</div>
-			<div class="span5 team">
-				 <a href="https://github.com/eclipse/xtext-xtend/issues" class="anchor-in-div" title="Bug reports and enhancement request are tracked at GitHub. Please explain the problem and provide a reduced but reproducable eample. Also please explain the concrete use case when requesting enhancements."></a>
-				 <div class="row">
-				    <div class="span1">
-				      <img src="{{ site.baseurl }}/images/bug-white.png" width="80" height="80" alt="image" class="image_left">
+			<div class="row">
+				<div class="span5 team">
+					<a href="https://groups.google.com/group/xtend-lang" class="anchor-in-div" title="The Xtend Google Group is the first source for getting answers in case you got stuck. The community is very friendly. Please ask any questions clearly and with sufficient information attached."></a>
```

Comment:
Wasn't gitter the first source?

Reply:
I do not know.

Reply:
nobody uses that group anymore. its just used from spam.
i'd prefer anything like the forum or stackoverflow over the d*mn google group

Reply:
@cdietrich : If you provide the stackoverflow/forum links as comment I will update the PR.

Reply:
?
https://www.eclipse.org/forums/index.php/f/27/
https://stackoverflow.com/

Reply:
Thanks. I was not sure whether there is dedicated channels to Xtend or the Xtext channels should be used.

Reply:
Done. @cdietrich Please review.